lib/models/villager_model.dart

```diff
@@ -0,0 +1,42 @@
+class villager {
```

Comment:
1. all classes should be in Pascal Case, please rename it
2. If this is a data model please mention so, make it as relevant as possible